--- a/config/BuildSystem/config/packages/sundials2.orig.py	2019-09-30 01:36:08.000000000 +0200
+++ b/config/BuildSystem/config/packages/sundials2.py	2020-04-04 16:15:52.074244560 +0200
@@ -3,8 +3,8 @@
 class Configure(config.package.GNUPackage):
   def __init__(self, framework):
     config.package.GNUPackage.__init__(self, framework)
-    self.version          = '2.5.0'
-    self.maxversion       = '2.5.100000'
+    self.version          = '5.2.0'
+    self.maxversion       = '5.7.000000'
     self.versionname      = 'SUNDIALS_PACKAGE_VERSION'
     self.download         = ['http://ftp.mcs.anl.gov/pub/petsc/externalpackages/sundials-'+self.version+'p1.tar.gz']
     self.downloaddirnames = ['sundials']
